docs/src/language_servers.md

```diff
@@ -540,6 +540,26 @@ Follow installation instructions on [LSP-typescript](https://github.com/sublimel
 
 Follow installation instructions on [LSP-vue](https://github.com/sublimelsp/LSP-vue).
 
+## Vala
+1. Install the [Vala-TMBundle](https://packagecontrol.io/packages/Vala-TMBundle) package from Package Control to add Vala syntax highlighting and for Vala files to be reconginsed.
```

Comment:
Please add an empty line between `## Vala` and the paragraph like the rest of the file.